### DISCUSSION TITLE
enum table column is empty and the selected status will not be displayed

### DISCUSSION THREAD

**Discussion by killjin** [filamentphp/filament#8807] [Help]
https://filamentphp.com/docs/3.x/support/enums

SelectColumn::make('status')
    ->options(Status::class)
   
![image](https://github.com/filamentphp/filament/assets/2804854/cb1780b5-bfe0-436b-9d39-f208b60b0507)

The table column is empty and the selected status will not be displayed.

**Comment by krekas** [comment]
how does your enum class look?
